<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CdoSampleAppWindow</class>
 <widget class="QMainWindow" name="CdoSampleAppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddLive sample application</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white</string>
    </property>
    <property name="title">
     <string>Video</string>
    </property>
    <widget class="RenderingWidget" name="localRenderer" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>160</width>
       <height>120</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:blue</string>
     </property>
    </widget>
    <widget class="RenderingWidget" name="remoteRenderer" native="true">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:green</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>251</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select cam:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="camCombo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Loading...</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select mic</string>
     </property>
    </widget>
    <widget class="QComboBox" name="micCombo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Loading...</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select speakers</string>
     </property>
    </widget>
    <widget class="QComboBox" name="spkCombo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Loading...</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="playTestSndBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Scope id:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="scopeIdTxt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>room1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnectBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="publishAudioChck">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Publish audio</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="publishVideoChck">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Publish video</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>AddLive SDK version:</string>
     </property>
    </widget>
    <widget class="QLabel" name="versionLbl">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>490</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Loading...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderingWidget</class>
   <extends>QWidget</extends>
   <header>renderingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
